### ISSUE TITLE
New UI contribution screen does not show Cancel button.

### ISSUE THREAD

**Issue by scorpsteals** [openequella/openEQUELLA#1157]
**Describe the bug**
When contributing to a new library the Cancel button is hidden along with no horizontal scroll option. 

**Expected behavior**
Cancel option should be available to click.

**Screenshots**
![image](https://user-images.githubusercontent.com/2992160/62634659-43cc9700-b904-11e9-866f-f4857a07c900.png)

![image](https://user-images.githubusercontent.com/2992160/62635038-ee44ba00-b904-11e9-9e58-3702cbff9e4a.png)

**Platform:**
- OpenEquella Version: [2019.1-Stable.r2.OSE]
- OS: [Windows 10]
- Browser [Chrome,firefox]

**Comment by scorpsteals** [comment]
Issue also exists when editing an existing item in a library.

**Comment by cathfitz** [comment]
Hi @scorpsteals - I've had a look at this issue, and can't reproduce in Chrome or FF. Do you have any further details you could provide? Thanks.

**Comment by scorpsteals** [comment]
So I tried Microsoft Edge and had the same issue show up. Tried contributing to a library with a different schema and I was able to see half of the Cancel button. 

![image](https://user-images.githubusercontent.com/2992160/62790093-52e44e00-ba98-11e9-94fe-d0ccfd3387bc.png)


Next I tried turning off the new UI and it renders fine on the old UI.

![image](https://user-images.githubusercontent.com/2992160/62789787-b0c46600-ba97-11e9-8c10-ffc4d126b3d8.png)

I am working on 2 monitors one with 1920x1080 and the other with 1280x1084 and both are showing the same issue.
